app/views/internal/configs/show.html.erb

```diff
@@ -888,13 +888,11 @@
             </tr>
           </thead>
           <tbody>
-            <% ENVied.config.variables.each do |v| %>
-            <% if v.group.in?(ENVied.required_groups) %>
-            <tr>
-              <td><%= v.name %></td>
-              <td style="max-width: 750px;"><%= ApplicationConfig[v.name] %></td>
-            </tr>
-            <% end %>
+            <% ENV.sort.each do |key, value| %>
```

Comment:
**question (non-blocking):** Just out of curiosity, will this list be much longer or were most values required anyway so there's barely a difference?

Reply:
We will double the keys we have. I do think this will be better however bc we will have a clear idea of ALL our ENV variables. There have been a couple that have popped up that I didn't realize we even had in Heorku. 
```
irb(main):001:0> ENVied.config.variables.count
=> 72
irb(main):002:0> ENV.keys.count
=> 168
```

Reply:
That makes sense, thanks for clarifying.